load_fixtures.py

```diff
@@ -0,0 +1,17 @@
+from os import chdir, getcwd, listdir, path, system
+
+original_directory = getcwd()
+apps = ['authentication', 'property', 'pages']
+files = []
+
+
+def loaddata(file):
+    if path.splitext(file)[1] == '.json' and file != 'initial_data.json':
+        system("python3 manage.py loaddata %s" % file)
+
+
+for app in apps:
+    files = listdir(f'{original_directory}/{app}/fixtures')
+    files.sort()
```

Comment:
Kindly clarify why you used the `sort()` function on the files list

Reply:
okay @Tevinthuku. I am using sort() in order to influence the flow of loading fixtures in the database; since we have some data which depend on others, we can use it to load the depended on fixtures earlier before the dependent fixtures are loaded by naming the fixtures using alphabetic order.